### PR TITLE
fix: use logger to throw error with proper exit code

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
fix

**Did you add tests for your changes?**
Nope
**If relevant, did you update the documentation?**
Nope
**Summary**
user logger for error

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Review by evilebottnawi — APPROVED** [review]

File: packages/webpack-cli/lib/utils/resolve-command.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const run = async (name, ...args) => {
             });
         } catch (err) {
             logger.error(`Action Interrupted, use ${cyan('webpack-cli help')} to see possible commands.`);
+            process.exit(2);
```
Comment (evilebottnawi) [inline packages/webpack-cli/lib/utils/resolve-command.js:20]: Can we add test

File: packages/webpack-cli/lib/utils/run-command.js [hunk 2]
```diff
@@ -7,7 +8,8 @@ async function runCommand(command, args = []) {
             shell: true,
         });
     } catch (e) {
-        throw new Error(e);
+        logger.error(e);
+        process.exit(2);
```
Comment (evilebottnawi) [inline packages/webpack-cli/lib/utils/run-command.js:12]: And test here too

File: packages/webpack-cli/lib/utils/resolve-command.js (hunk 1 above)
Comment (snitin315) [reply packages/webpack-cli/lib/utils/resolve-command.js:20]: I'm not sure how to write a for this

Comment (evilebottnawi) [reply packages/webpack-cli/lib/utils/resolve-command.js:20]: unit test inside package, just run `resolve('unknown')`

**Review by evilebottnawi — APPROVED** [review]
Looks good

**Review by anshumanv — APPROVED** [review]
looks good

**Pull request merged**